src/Polly.Shared/PolicyAsync.cs

```diff
@@ -39,14 +50,25 @@ public Task ExecuteAsync(Func<Task> action)
         /// </summary>
         /// <param name="action">The action to perform.</param>
         [DebuggerStepThrough]
-        public async Task<PolicyResult> ExecuteAndCaptureAsync(Func<Task> action)
+        public Task<PolicyResult> ExecuteAndCaptureAsync(Func<Task> action)
+        {
+            return ExecuteAndCaptureAsync(false, action);
+        }
+
+        /// <summary>
+        ///     Executes the specified asynchronous action within the policy and returns the captured result.
+        /// </summary>
+        /// <param name="continueOnCapturedContext">Whether to continue on a captured synchronization context.</param>
+        /// <param name="action">The action to perform.</param>
+        [DebuggerStepThrough]
+        public async Task<PolicyResult> ExecuteAndCaptureAsync(bool continueOnCapturedContext, Func<Task> action)
```

Comment:
Request move `bool continueOnCapturedContext` to be last parameter, in `ExecuteAndCaptureAsync()`